sdk/python/feast/feature_store.py

```diff
@@ -266,6 +273,10 @@ def get_historical_features(
                 SQL query. The query must be of a format supported by the configured offline store (e.g., BigQuery)
             feature_refs: A list of features that should be retrieved from the offline store. Feature references are of
                 the format "feature_view:feature", e.g., "customer_fv:daily_transactions".
+            full_feature_names: By default, this value is set to False. This strips the feature view prefixes from the data
```

Comment:
Should be - "This adds the feature view name to prefix the feature name."

Reply:
some of this still refers to the old feature_name_only parameter from the previous version